v2/internal/controllers/controller_resources.go

```diff
@@ -132,9 +132,17 @@ func getGeneratedStorageTypes(
 
 	// Verify we're using the hub version of VirtualNetworksSubnet in the loop below
 	var _ ctrlconversion.Hub = &networkstorage.VirtualNetworksSubnet{}
+	var _ ctrlconversion.Hub = &networkstorage.LoadBalancersInboundNatRule{}
+	var _ ctrlconversion.Hub = &networkstorage.RouteTablesRoute{}
```

Comment:
minor: Why is this line (136) needed here?

Reply:
Hub version verification was missing for `RouteTablesRoute` so just added that here